packages/embed-widget/src/index.scss

```diff
@@ -1,14 +1 @@
 @import '@deephaven/components/scss/custom.scss';
-
-body {
```

Comment:
You can just remove this file. This doesn't cause any style changes between the app and embed-widget?